edsby.py

```diff
@@ -515,39 +615,52 @@ def getClassAssignmentList(self, classNID, classRID):
             assignmentNID = str(metadata[assg]['nid'])
 
             # Copy other keys from metadata obj to compiled assignment data obj
-            for key in metadata['r'+str(metadata[assg]['rid'])]:
-                if assignmentNID in assignmentData['assignments']: # If we've found metadata for an asssignment that we've also found scores for
-                    assignmentData['assignments'][assignmentNID][key] = metadata[assg][key] # Copy metadata to that entry
+            for key in metadata['r' + str(metadata[assg]['rid'])]:
+                if assignmentNID in assignmentData[
```

Comment:
Here's another example of PyCharm mangling our formatting: 

```python
                if assignmentNID in assignmentData[
                     'assignments']:  # If we've found metadata for an asssignment that we've also found scores for
                     assignmentData['assignments'][assignmentNID][key] = metadata[assg][
                         key]  # Copy metadata to that entry
```

There's no real gain from splitting up these lines, and it makes the code much more difficult to follow along with.